llvm/include/llvm/SYCLLowerIR/SYCLPropagateAspectsUsage.h

```diff
@@ -14,13 +14,22 @@
 #define LLVM_SYCL_PROPAGATE_ASPECTS_USAGE_H
 
 #include "llvm/IR/PassManager.h"
+#include "llvm/Support/CommandLine.h"
 
 namespace llvm {
 
+struct SPAUOptions {
+  SmallVector<std::string, 8> Targets;
+};
+
 class SYCLPropagateAspectsUsagePass
     : public PassInfoMixin<SYCLPropagateAspectsUsagePass> {
 public:
+  SYCLPropagateAspectsUsagePass(SPAUOptions Opts = {}) : Opts(Opts){};
```

Comment:
So, that would mean that clang FE will also have to construct this struct to pass the argument: will it be better to accept a plain string here and do the parsing internally. This way, we would avoid parsing at FE level